health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/SearchAttendeeBoundary.js

```
@@ -0,0 +1,305 @@
+        return boundaryIndex <= lowestLevelIndex;
+    };
+
+    return (
+        <React.Fragment>
+            <div>
+                <span
+                    onClick={() => {
+                        setReset(true);
+                    }}
+                    style={{ border: "1px solid #e0e0e0", padding: "6px", marginBottom: "10px", cursor: "pointer" }}
+                >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)